fonts.css

```diff
@@ -0,0 +1,38 @@
+@font-face {
+    font-family: 'PT Serif';
+    src: url('../font/PTSerif-Regular.eot');
+    src: url('../font/PTSerif-Regular.eot?#iefix') format('embedded-opentype'),
+        url('../font/PTSerif-Regular.woff2') format('woff2'),
+        url('../font/PTSerif-Regular.woff') format('woff'),
+        url('../font/PTSerif-Regular.ttf') format('truetype'),
+        url('../font/PTSerif-Regular.svg#PTSerif-Regular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'PT Serif';
+    src: url('../font/PTSerif-Bold.eot');
+    src: url('../font/PTSerif-Bold.eot?#iefix') format('embedded-opentype'),
+        url('../font/PTSerif-Bold.woff2') format('woff2'),
+        url('../font/PTSerif-Bold.woff') format('woff'),
+        url('../font/PTSerif-Bold.ttf') format('truetype'),
+        url('../font/PTSerif-Bold.svg#PTSerif-Bold') format('svg');
+    font-weight: bold;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Museo Sans Cyrl 500';
+    src: url('../font/MuseoSansCyrl-500.eot');
+    src: url('../font/MuseoSansCyrl-500.eot?#iefix') format('embedded-opentype'),
+        url('../font/MuseoSansCyrl-500.woff2') format('woff2'),
+        url('../font/MuseoSansCyrl-500.woff') format('woff'),
+        url('../font/MuseoSansCyrl-500.ttf') format('truetype'),
+        url('../font/MuseoSansCyrl-500.svg#MuseoSansCyrl-500') format('svg');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5764748)